src/components/global/CursorPagination.vue

```diff
@@ -0,0 +1,40 @@
+<script setup>
+import { defineProps } from "vue";
+
+const props = defineProps({
+  pageInfo: {
+    type: Object,
+    default: null,
+  },
+  onPageChange: Function,
+});
+</script>
+
+<template>
+  <div
+    v-if="props.pageInfo"
+    class="text-center border-t-thin py-6 d-flex justify-center align-center ga-2"
+  >
+    <v-btn
+      :disabled="!props.pageInfo.hasPreviousPage"
+      @click="props.onPageChange('startCursor')"
+      icon="mdi-menu-left"
+      color="primary"
+      size="small"
+      variant="tonal"
+    >
+    </v-btn>
+    <v-icon color="primary" size="small" class="opacity-20"
+      >mdi-dots-horizontal</v-icon
```

Comment:
I guess we would have two options here:
- not show anything at all (removing the 3 dots as they don't offer any information)
- showing "Page 1", "Page 2" and tracking the current page in the frontend (and always starting from Page 1)

With the second approach, if the user clicks a few times to the right, they at least know how "deep" they went into the history (even if not knowing how many pages there are in total)